content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,241 @@
+location, or for an audience that isn't the API server, use one of the
+following methods:
+
+* [TokenRequest API](/docs/reference/kubernetes-api/authentication-resources/token-request-v1/)
+  (recommended): Request a short-lived service account token using your
+  *application code*. The token expires automatically and can rotate upon
+  expiration.
+* [Token Volume Projection](/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)
+  (recommended): In Kubernetes v1.20 and later, use the Pod specification to
+  tell the kubelet to add the service account token to the Pod as a
+  *projected volume*. Projected tokens expire automatically, and the kubelet
+  rotates the token before it expires. The ServiceAccount token is also
+  invalidated in the Kubernetes API when the Pod or the ServiceAccount is
+  deleted.
+* [Service Account Token Secrets](/docs/tasks/configure-pod-container/configure-service-account/#manually-create-a-service-account-api-token)
+  (not recommended): You can mount service account tokens as Kubernetes
+  Secrets in Pods. These tokens don't expire and don't rotate. This method
+  is not recommended, especially at scale, because of the risks associated
+  with static, long-lived credentials. In Kubernetes v1.24 and later, the
+  [LegacyServiceAccountTokenNoAutoGeneration feature gate](/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-graduated-or-deprecated-features)
+  prevents Kubernetes from automatically creating these tokens for
+  ServiceAccounts.
+
+## Authenticating service account credentials {#authenticating-credentials}
+
+ServiceAccounts use signed
+{{<glossary_tooltip term_id="jwt" text="JSON Web Tokens">}}  (JWTs)
+to authenticate to the Kubernetes API server, and to any other system where a
+trust relationship exists. Depending on the token issuer, a ServiceAccount
+token might also have an expiry time, an audience, and a time after which the
+token *starts* being valid. When a process running in a Pod attempts to
+communicate with the Kubernetes API server, it adds an
+`Authorization: Bearer <token>` header to the HTTP request. The API server
+checks the validity of the bearer token as follows:
+
+1. Check the token signature.
+1. Check whether the token has expired.
+1. Check whether the token is currently valid.
+1. Check the audience claims.
+
+For tokens issued using the `TokenRequest` API, the API server also checks that
+the specific Pod using the ServiceAccount still exists, matching by the
+{{< glossary_tooltip term_id="uid" text="unique ID" >}} of that Pod. For legacy
+tokens that are mounted as Secrets in Pods, the API server checks the token
+against the Secret.
+
+For more information about the authentication process, refer to
+[Authentication](/docs/reference/access-authn-authz/authentication/#service-account-tokens).
+
+### Authenticating service account credentials in your own code {authenticating-in-code}
```

Comment:
This section definitely should mention token audiences.

Reply:
Could you suggest a line or two? We want them to specify intended audiences for tokens obtained using TokenRequest so that the token authenticator can validate that the recipient is in the list of audiences/

Reply:
Wrapped, that becomes
```markdown
Your application should always define its own audience that it is going to accept, and it should
check that the token's audiences match the audiences the application expects. This is to help
minimize the scope of the token, making it only usable in your application and nowhere else.
```

Reply:
Done